phyton.py

```diff
@@ -0,0 +1,7 @@
+import time
+
+print("Program is calculating, please wait...")
+
+while True:
+       time.sleep(1)
```

Comment:
Solid changes, thats a great use of sleep

Reply:
nah, it's bloated, had to import whole library for that

Reply:
Yes, but I jest looked into logs, and we got 3 times faster response time, and saved up some space in cloud. Dont be so strict on yourself Krzysztof. Kudos to you, great work!!